### PR TITLE
Update readme test

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Rework `Testing` section in README

* move 'Test inhibition labels' to 'Alertmanager inhibition dependency check'
* change 'make test' to 'make test-rules'
* add global 'Testing' section and 'Prometheus rules unit tests'

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -269,3 +269,21 @@ Those rules can be written according to this template :
...
+  - an Alert rule with some target labels
+
+This is what we call the inhibition dependency chain.
+
+One can check whether inhibition labels (mostly "cancel_if_" prefixed ones) are well defined and triggered by a corresponding label in the alerting rules by running the `make test-inhibitions` command at the projet's root directory.
+
+This command will output the list of missing labels. Each of them will need to be defined in the alerting rules.
+If there is no labels outputed, this means tests passed and did not find missing inhibition labels.
+
+Warning: the tool may output false alerts or miss some alerts because of the following limitations.
```
Comment (hervenicol) [inline README.md:287]: I think the "limitations" part is outdated. @QuantumEnigmaa , maybe you want to change that part?

**Review by hervenicol — APPROVED** [review]
I like it :)

File: README.md (hunk 1 above)
Comment (QuantumEnigmaa) [reply README.md:285]: I think the second assertion can be considered false. However I'm not sure about the 1st one : the script definitely check labels for each provider but I still have to check whether it throws errors if an inhibition is missing for a specific provider.

Comment (TheoBrigitte) [reply README.md:285]: merging this. feel free to update when needed.

**Pull request merged**